### ISSUE TITLE
Network offerings dropdown is broken in UI

### ISSUE THREAD

**Issue by PaulAngus** [apache/cloudstack#3487]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
Master
~~~

##### OS / ENVIRONMENT

NA

##### SUMMARY

When selecting an Isolated Network Offering, numerous offerings are listed that are a. not relevant and b. repeated

##### STEPS TO REPRODUCE

~~~
(not sure if relevant yet but) created a custom VPC offering limited to a sub domain and current zone.
(not sure if relevant yet but) created a custom network offering for VPC limited to a sub domain and current zone.

Then went to create an isolated as a user in the parent domain.
~~~

##### EXPECTED RESULTS

~~~
'normal' entries in drop down were expected
~~~

##### ACTUAL RESULTS

~~~
![image](https://user-images.githubusercontent.com/4810220/61084149-43d09880-a425-11e9-9346-7b4ce0bc6b06.png)
~~~

**Comment by PaulAngus** [comment]
cc @shwstppr @borisstoyanov @anuragaw

**Comment by shwstppr** [comment]
I don't think this issue is related to master(tested locally) and is only with https://github.com/apache/cloudstack/pull/3248.
It is due to adding multiple dependsOn for networkOfferingId UI element, it calls the same method twice instantly and populates offerings twice. https://github.com/apache/cloudstack/pull/3248/files?file-filters%5B%5D=.js#diff-d617598675f6e8d23deab5bd5e3309f7R560
I'm fixing that. This issue can be closed if not reproducible by others on master.
@PaulAngus @rhtyd

**Comment by rhtyd** [comment]
Okay @shwstppr please send a PR. Thanks.

**Comment by PaulAngus** [comment]
thanks I wasn't sure where it came from (which is why i CC'd every man and his dog), but would end up in master if 3248 was merged anyway

**Comment by anuragaw** [comment]
I cannot repro this on master @PaulAngus , @rhtyd , @shwstppr . Since the PR #3248 is not merged we can perhaps close this issue?

**Comment by rhtyd** [comment]
Thanks @anuragaw, let's fix it in #3248 if not done already (cc @shwstppr). Closing this on testing remark.
